docs/src/pages/index.js

```diff
@@ -101,11 +101,15 @@ export default function Home() {
       .fetch("https://api.github.com/repos/nextauthjs/next-auth")
       .then((res) => res.json())
       .then((data) => {
+        const stargazersCount = data?.stargazers_count
+
+        if (typeof stargazersCount !== 'number' || !isFinite(stargazersCount)) return
+
         const githubLink = document.querySelector(
           ".navbar__item.navbar__link[href*='github']"
         )
         const githubStat = document.createElement("span")
-        githubStat.innerHTML = kFormatter(data.stargazers_count)
+        githubStat.innerHTML = kFormatter(stargazersCount)
```

Comment:
How about just somethign like this? The number isn't going to sway much in the short-term, so I think its safe to fall back to this:

```suggestion
        githubStat.innerHTML = kFormatter(stargazersCount ?? '21000')
```